context_test.go

```diff
@@ -58,10 +58,16 @@ func TestSplitAbsoluteProjectRoot(t *testing.T) {
 		}
 	}
 
-	// test where it should return error
-	got, err := depCtx.SplitAbsoluteProjectRoot("tra/la/la/la")
+	// test where it should return an error when directly within $GOPATH/src
+	got, err := depCtx.SplitAbsoluteProjectRoot(filepath.Join(depCtx.GOPATH, "src"))
 	if err == nil {
```

Comment:
nit: let's do at least a cursory check for the error text we expect - e.g., `if err == nil || strings.Contains(err.Error(), "$GOPATH/src") {`, to give us a bit of an assurance that we're getting the error we expect to.